packages/pmml-editor/src/editor/components/Outputs/organisms/OutputFieldsTable.tsx

```diff
@@ -130,7 +131,7 @@ export const OutputFieldsTable = (props: OutputFieldsTableProps) => {
             className={`output-item output-item-n${activeOutputFieldIndex} editable ${
               activeOutputFieldIndex === index ? "editing" : ""
             }`}
-            key={o.name.value}
+            key={o.name as string}
```

Comment:
For some reason, what we get for names are plain strings, not `FieldName`'s. Had to change this to avoid the `elements in a list need a unique key` console react error.